crates/nu-command/src/plot/plot_rose.rs

```diff
@@ -0,0 +1,69 @@
+use super::rose_::create_plot;
+use nu_engine::CallExt;
+use nu_protocol::{
+    ast::Call,
+    engine::{Command, EngineState, Stack},
+    Category, Example, PipelineData, ShellError, Signature, SyntaxShape, Type, Value,
+};
+
+#[derive(Clone)]
+pub struct RosePlot;
+
+impl Command for RosePlot {
+    fn name(&self) -> &str {
+        "plot rose"
+    }
+
+    fn usage(&self) -> &str {
+        "Create and display a rose plot from pipeline data."
+    }
+    fn signature(&self) -> Signature {
+        Signature::build(self.name())
+            .input_output_type(Type::Any, Type::Any)
+            .category(Category::Custom("plotting".into()))
+            .required(
+                "labels",
+                SyntaxShape::String,
+                "category field for the rose plot",
+            )
+            .required("values", SyntaxShape::Any, "value field for the rose plot")
+    }
+    fn examples(&self) -> Vec<Example> {
+        vec![Example {
+            description: "creates a rose plot from Nu data",
+            example: "let a = ls; | plot rose $a.type $a.size",
```

Comment:
early feeback, but this is a bit confusing i must admit :open_mouth: 
i would write either
```nushell
let a = ls; plot rose $a.type $a.size
```
or
```nushell
ls | plot rose $in.type $in.size
```
:yum:

Reply:
Ah I didn't know you could do the second one! I'll change that!